media-autobuild_suite.bat

```diff
@@ -1337,49 +1337,39 @@ if not exist %instdir%\mintty.lnk (
     echo -------------------------------------------------------------------------------
     echo.- make a first run
     echo -------------------------------------------------------------------------------
-    call :runBash firstrun.log "-c exit"
+    call :runBash firstrun.log exit
+
+    sed -i "s/#Color/Color/;s/^^IgnorePkg.*/#&/" %instdir%\%msys2%\etc\pacman.conf
```

Comment:
Is the color thing needed? You could always add `--color <when>` to select pacman commands if you really want to add colors.

Reply:
It's not really, just a convenience thing when using the mintty lnk outside of the script

Reply:
I usually add it myself, but sure, you can leave it. The logs are pruned of color characters anyway.